src/components/pages/books/books.css

```diff
@@ -185,3 +185,41 @@
 .addBtn:hover {
   cursor: pointer;
 }
+
+@media screen and (max-width: 768px) {
+  .book-section-one {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
```

Comment:
| Zeplin's Design|
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/1d46b7ec-613b-4958-8db3-19eabe5a01f9) |

| @AleWaweru Design |
|-|
| ![image](https://github.com/AleWaweru/Bookstore/assets/93039359/4e733e38-50f3-40f4-9a67-47b9c6a75f92) |

- [ ] It would be nice if you double-check the colors of the buttons and the logo because they are not following Zeplin's color scheme 😨 Kindly follow Zeplin's design to make your design more descriptive 👍
- [ ] It would be nice if you make your navbar background color white to follow the project requirements :+1:
- [ ] It would be nice if you make your user icon in the center of the circle to make your app look attractive :+1:

Reply:
done.